src/msw/graphicsd2d.cpp

```diff
@@ -423,6 +598,14 @@ IDWriteFactory* wxDWriteFactory()
             wxIID_IDWriteFactory,
             reinterpret_cast<IUnknown**>(&gs_IDWriteFactory)
             );
+#if wxUSE_PRIVATE_FONTS
+        // Register our custom font loader
+        HRESULT hr = gs_IDWriteFactory->RegisterFontCollectionLoader(wxDirect2DFontCollectionLoader::GetLoader());
+        if ( FAILED(hr) )
+        {
+            wxLogError("Could not register custom DirectWrite font loader.");
```

Comment:
Sorry for continued nitpicking with error handling, but this should be
```suggestion
            wxLogError(_("Could not register custom DirectWrite font loader."));
```
as it's a user-visible string.